[2022-06-13 18:00:08,611] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: listing_4_05.get_data scheduled__2022-06-13T17:00:00+00:00 [queued]>
[2022-06-13 18:00:08,624] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: listing_4_05.get_data scheduled__2022-06-13T17:00:00+00:00 [queued]>
[2022-06-13 18:00:08,627] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 18:00:08,628] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-13 18:00:08,630] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 18:00:08,640] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_data> on 2022-06-13 17:00:00+00:00
[2022-06-13 18:00:08,646] {standard_task_runner.py:52} INFO - Started process 37485 to run task
[2022-06-13 18:00:08,649] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_05', 'get_data', 'scheduled__2022-06-13T17:00:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_5.py', '--cfg-path', '/tmp/tmpv7cq899a', '--error-file', '/tmp/tmp8q7woxpo']
[2022-06-13 18:00:08,650] {standard_task_runner.py:80} INFO - Job 132: Subtask get_data
[2022-06-13 18:00:08,703] {task_command.py:369} INFO - Running <TaskInstance: listing_4_05.get_data scheduled__2022-06-13T17:00:00+00:00 [running]> on host 48de8cc79e8d
[2022-06-13 18:00:08,762] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_05
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-13T17:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-13T17:00:00+00:00
[2022-06-13 18:00:08,765] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-06-13 18:01:07,145] {python.py:173} INFO - Done. Returned value was: None
[2022-06-13 18:01:07,156] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=listing_4_05, task_id=get_data, execution_date=20220613T170000, start_date=20220613T180008, end_date=20220613T180107
[2022-06-13 18:01:07,230] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-13 18:01:07,247] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
